### PR TITLE
Added Express server boilerplate

### PR DESCRIPTION
**Pull request by kjschu**
Added an example Express.js server

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 5000;
+
+const bodyParser = require('body-parser');
```
Comment (lindlym) [inline server.js:5]: Dude, do we even need body parser?

File: server.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const cookieParser = require('cookie-parser');
+const cors = require('cors');
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(cookieParser());
+app.use(cors());
```
Comment (lindlym) [inline server.js:12]: Nice grouping of your middleware here.

File: server.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+app.get('/', (req, res) => {
+  res.status(200);
+  res.send('Receiving data...');
```
Comment (lindlym) [inline server.js]: You can chain these.

`res.status(200).send('Receiving data...');`

**Review by lindlym — CHANGES_REQUESTED** [review]
Go ahead and fix the comments that need fixing before I will merge this in, but nice work on this overall!

File: server.js (hunk 1 above)
Comment (kjschu) [reply server.js:5]: Yes, we're using it to handle requests

File: server.js (hunk 3 above)
Comment (kjschu) [reply server.js]: Fixed

**Comment by lindlym** [comment]
He fixed my changes. Whoever leaves the last comment on changes needing to be made, just merge in unless more changes need to be made.

**Pull request merged**